tests/test_ansi2html.py

```diff
@@ -107,10 +107,10 @@ def test_conversion_as_command(
         with open(join(_here, "ansicolor.html"), "rb") as output:
             expected_data = "".join(read_to_unicode(output))
 
-        def f() -> StringIO:
-            return StringIO(test_data)
-
-        with patch("sys.stdin", new_callable=f):
+        with patch(
+            "sys.stdin",
+            new_callable=lambda: TextIOWrapper(BytesIO(test_data.encode("utf-8"))),
```

Comment:
The four uses of `new_callable` in here could probably be simplified from `new_callable=lambda: X` to `new=X` to `X`. Am I missing something?

Reply:
Yes, I'll change that.

Reply:
I fixed it.

Reply:
Thank you!